### ISSUE TITLE
yq does not recognise unclosed with( / unclosed with( with --inplace clear the file contents

### ISSUE THREAD

**Issue by landure** [mikefarah/yq#1366]
When using with(), and not closing the parenthesis, yq does not throw an error, and try to apply the command. This result in a empty file.

Version of yq: 4.27.5
Operating system: Ubuntu 22.04 Jammy Jellyfish
Installed via: binary release

**Input file**

docker-compose.yml:
```yaml
services:
  beehive: {}
```
**Command**

```bash
command yq --inplace 'eval' 'with(.services.beehive.restart;
    . |= "unless-stopped"
    | . line_comment="Restart the service on failure."' \
    docker-compose.yml"```
```

**Actual behavior**

```yaml
# empty file #
```

**Expected behavior**

```bash
Syntax Error: unclosed with.
```

**Comment by mikefarah** [comment]
Thanks - will fix in the next release

**Comment by mikefarah** [comment]
Fixed in 4.28.2
